lib/watcher.js

```diff
@@ -51,6 +56,7 @@ Watcher.prototype.run = function () {
                 // otherwise run remaining pending tests
                 if (!Object.keys(self._runners).length) {
                     self.emit('end');
+                    process.exit(failed);
```

Comment:
Would it be better if you check `failed` here and if it's more than 0, exit with 1, else 0?
Exit codes tend to have [their own meanings](http://tldp.org/LDP/abs/html/exitcodes.html), and by using this approach exit code for failing tests would depend on suites number which is quite weird.

Btw, I think the overall behaviour should be consistent with mocha, so it would also be nice if you write some test checking this behaviour.